# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 2adbab502a1e
# Full Hash: 2adbab502a1e690fccd61fb604be820e60d61723
# Author: Kristen Wright <kwright@mozilla.com>
# Date: 2020-09-09 09:39:57
# Regressor Bug: 1642727
# File Overlap Count: 1
# Description:
#   Bug 1642727 - Remove the VarCache machinery r=njn
#   
#   This commit removes Varcache entirely, putting an end to our quest to get rid of VarCache.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79538
# ==============================================================================

diff -r 1aeb7707569f -r 2adbab502a1e modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Sep 08 15:05:06 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Sep 08 17:12:01 2020 +0000
@@ -2180,7 +2180,7 @@
 # Whether we disable triggering mutation events for changes to style
 # attribute via CSSOM.
 # NOTE: This preference is used in unit tests. If it is removed or its default
-# value changes, please update test_sharedMap_var_caches.js accordingly.
+# value changes, please update test_sharedMap_static_prefs.js accordingly.
 - name: dom.mutation-events.cssom.disabled
   type: bool
   value: true
@@ -3214,7 +3214,7 @@
   mirror: always
 
 # NOTE: This preference is used in unit tests. If it is removed or its default
-# value changes, please update test_sharedMap_var_caches.js accordingly.
+# value changes, please update test_sharedMap_static_prefs.js accordingly.
 - name: dom.webcomponents.shadowdom.report_usage
   type: bool
   value: false